test/file.ts

```diff
@@ -1892,6 +1902,28 @@ describe('File', () => {
       writable.write('data');
     });
 
+    describe('should succeed if resumable requested but config dir doesnt exist', () => {
+      const configDir = xdgBasedirCached.config;
+
+      before(() => {
+        rimraf.sync(configDir);
+      });
+
+      it('should succeed if resumable requested but config dir doesnt exist', done => {
+        assert.strictEqual(fs.existsSync(configDir), false);
```

Comment:
`fs.access(configDir, fs.constants.W_OK, accessErr => {` is not throwing an error for this test and I'm really confused as to why. @stephenplusplus any ideas?

Reply:
I believe that was because we were stubbing `fs.access` earlier in the test file to always pass. Checking into this caused me to get in the editor and write a bit of code as I worked through the issue, and I ended up with a few tests to cover the changes in the PR. I'm sorry I keep doing this. It seems to work out that when I'm investigating an issue, I end up writing a bunch of code to test my theories out. So, here is what I ended up with: https://gist.github.com/stephenplusplus/dccf237542b7a702ebff90f03f2a557a. Don't worry about using any of it if you wanted to go a different way with the tests.

Reply:
No need to apologize! Thanks so much for doing this!!! Super super helpful :) Really appreciate it

Reply:
Good catch noting that `fs.access` was being stubbed. I totally missed that.